src/app/functions/server/onerep.ts

```diff
@@ -479,25 +438,16 @@ export async function getProfilesStats(
   if (profileStatsCache.has(queryParamsString))
     return profileStatsCache.get(queryParamsString);
 
-  const response: Response = await onerepFetch(
+  const profileStats = (await onerepFetch(
     `/stats/profiles?${queryParamsString}`,
     {
       method: "GET",
     },
-  );
-  if (!response.ok) {
-    logger.error(
-      `Failed to fetch OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-    throw new Error(
-      `Failed to fetch OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-  }
-
-  const profileStats: ProfileStats = await response.json();
+  )) as Promise<ProfileStats>;
```

Comment:
I think this type assertion is incorrect (because the return value is `await`'ed), and so is the `await` on line 450.

```suggestion
  )) as ProfileStats;
```